pr_agent/tools/pr_reviewer.py

```diff
@@ -103,4 +108,37 @@ def _prepare_pr_review(self) -> str:
 
         if settings.config.verbosity_level >= 2:
             logging.info(f"Markdown response:\n{markdown_text}")
-        return markdown_text
\ No newline at end of file
+        return markdown_text
+
+    def _publish_inline_code_comments(self):
+        if settings.config.git_provider != 'github': # inline comments are currently only supported for github
+            return
+
+        review = self.prediction.strip()
+        try:
```

Comment:
Consider adding error handling for the case when the JSON parsing fails in '_publish_inline_code_comments' method. This will make the code more robust. [important]